### ISSUE TITLE
Raw inline HTML incorrectly processed

### ISSUE THREAD

**Issue by magicus** [atlassian/commonmark-java#25]
Raw HTML is not handled according to the spec. Example markdown:
```
test raw html:
<a><bab><c2c>
```

Actual output from commonmark-java:
```
<p>test raw html:</p>
<p>&lt;a&gt;&lt;bab&gt;&lt;c2c&gt;</p>
```

Expected output:
```
<p>test raw html:</p>
<a><bab><c2c>
```

See http://spec.commonmark.org/0.22/#example-559

**Comment by robinst** [comment]
What options are you setting on the `HtmlRenderer`? More specifically, do you use `escapeHtml(true)` (see [Javadoc](http://static.javadoc.io/com.atlassian.commonmark/commonmark/0.2.0/org/commonmark/html/HtmlRenderer.Builder.html#escapeHtml-boolean-))?

**Comment by magicus** [comment]
Yep, I did. User error. 

However, I started by copying and pasting the demo code on the front page. I see now when reading more carefully that "This uses the parser and renderer with default options" was continued by "... except for escaping raw HTML tags and blocks.".

I'll close this issue. However, you might want to consider display a simple first example that parses in commonmark compliant mode, for those of us with too little time to dig into the documentation and just wants to see if commonmark-java fits the bill.
